templates/flutter/.travis.yml.twig

```diff
@@ -5,6 +5,11 @@ dart: stable
 os: linux
 
 install:
+- cd ~
+- git clone https://github.com/flutter/flutter.git -b stable --depth 1
+- export PATH="$PATH:`pwd`/flutter/bin/cache/dart-sdk/bin"
```

Comment:
Let's use $() instead of `` for command substitution

Reply:
Updated